includes/admin/class-wc-admin-importers.php

```diff
@@ -177,6 +187,48 @@ public function post_importer_compatibility() {
 			}
 		}
 	}
+
+	/**
+	 * Ajax callback for importing one batch of products from a CSV.
+	 */
+	public function do_ajax_product_import() {
+		check_ajax_referer( 'wc-product-import', 'security' );
+
+		if ( ! current_user_can( 'edit_products' ) || ! isset( $_POST['file'] ) ) {
+			wp_die( -1 );
+		}
+
+		include_once( WC_ABSPATH . 'includes/import/class-wc-product-csv-importer.php' );
+
+		$file = $_POST['file'];
```

Comment:
Is this cool? I don't know if not doing anything to `$_POST['file']` opens the call to some sort of attack. I can't see how it would, but someone else might. Let me know if it does.